src/static/js/movements.js

```diff
@@ -49,6 +49,8 @@ $(function() {
                     { json_field: "ISTRIAL", post_field: "trial", label: _("Trial Adoption"), tooltip: _("Is this a trial adoption?"), type: "check" },
                     { json_field: "TRIALENDDATE", post_field: "trialenddate", label: _("Trial ends on"), tooltip: _("The date the trial adoption is over"), type: "date" },
                     { json_field: "COMMENTS", post_field: "comments", label: _("Comments"), type: "textarea" },
+                    { json_field: "EVENTLINK", post_field: "eventlink", label: _("Link to event"), type: "check", hideif: function(){return !common.has_permission("lem");}},
```

Comment:
hideif should also check a config switch to check whether events are enabled and have them off by default (add to configuration.py default map as EnableEvents: No). You can set EnableEvents to Yes in your configuration table manually while working on the functionality and we can merge it into master without anyone seeing it until you add the option to Settings->Options->Remove (if this makes sense)

Reply:
I added this functionality, and fixed a bug that I found in my code, hope it works well :-)
I changed it to DisableEvents: Yes in the default map and also added it already to settings.